app/scicrunch_process_results.py

```diff
@@ -24,7 +24,7 @@ def _prepare_results(results):
         attr['took'] = results['took']
         attr.update(sort_files_by_mime_type(attr['files']))
 
-        output.append(_manipulate_attr(attr))
```

Comment:
I think for now I would prefer to comment out this line because I would revisit this when annotations are improved for scaffolds.

Reply:
I will fix up the incorrect string labels after this is pulled in.

Reply:
Sure, done.